src/main/java/org/launchcode/liftoffrecipetracker/controllers/BeverageController.java

```diff
@@ -49,6 +54,9 @@ public String processCreateBeverageForm(@Valid @ModelAttribute Beverage beverage
                 return "beverage/create";
             }
 
+            User user = authenticationController.getUserFromSession(userSession);
+            beverage.setUser(user);
+
```

Comment:
User is not set when a beverage is created through the copy feature

Reply:
Updated this to include ownership here